scripts/ProcessTemplates.ps1

```diff
@@ -203,16 +207,22 @@ function Copy-Object {
 # CloneAndPullLocalizedRepos
 #----------------------------------------------------------------------------
 Function CloneAndPullLocalizedRepos {
-    $rootPath = Convert-Path "..\.."
+    # repros will be downloaded to .\scripts folder
+    # to make this run like pipeline build, we'll only clone the repo and not due a pul
+    # for testing, delete the repos each time
+    $rootPath = $localizeRoot
 
     Push-Location
     foreach ($lang in $supportedLanguages) {
+        if ($lang -eq $defaultLanguage) {
+            continue;
+        }
         $repoName = "$repoBaseName.$lang"
         $repoPath = "$rootPath\$repoName"
         if (Test-Path $repoPath) {
-            Write-Host "Repo exist on disk, performing git pull ..."
-            Set-Location -Path $repoPath
-            git pull
+            Write-Host "Repo exist on disk, skipping $repoPath ..."
+            #Set-Location -Path $repoPath
+            #git pull

```

Comment:
isn't this whole section of testing the path and doing pull or clone not needed now? didn't the restore script at the top do all this work already?

Reply:
this would help when testing locally to clone and the pull can be added back if we can determine if it's running in the pipeline or not